app-editors/bluefish/patches/bluefish-2.2.13.patchset

```diff
@@ -1,4 +1,4 @@
-From 8516485d77b7418ad1e44def1189f16966807e16 Mon Sep 17 00:00:00 2001
+From 44acc677ba81fcfee8469fade3f3852799c3dfec Mon Sep 17 00:00:00 2001
 From: begasus <begasus@gmail.com>
 Date: Mon, 21 Nov 2022 10:44:03 +0100
 Subject: Dirty hack for XDG variables
```

Comment:
Thanks, so this was already included for R1B4?

Reply:
Thanks! Fixed in: https://github.com/haikuports/haikuports/pull/8653

Reply:
> Just a heads up @Begasus:
> 
> This "dirty hack" shouldn't be necessary anymore, since [this commit](https://cgit.haiku-os.org/haiku/commit/?id=3af8011358bd4c624a0979336d48dabb466171ed) on mid Dicember last year (right before the last beta4 RC).

@OscarL  Is there a reason as to why the `/etc/profile.d/xdg_basedirs.sh` file is left behind?, is it harmless?